src/main/java/colress/TaskList.java

```diff
@@ -0,0 +1,146 @@
+package colress;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import colress.task.Task;
+
+/**
+ * Represents the TaskList of the Colress chatbot.
+ */
+public final class TaskList {
+    private final ArrayList<Task> TASKS;
```

Comment:
Is there a reason TASKS is in all caps?

Reply:
Changed to lowercase. I wrongly characterised the variables as constants .Thanks for the feedback👍.